Detections/SigninLogs/Sign-in Burst from Multiple Locations.yaml

```diff
@@ -24,6 +24,6 @@ query: |
   | where ResultType == 0
   | summarize CountOfLocations = dcount(Location), Locations = make_set(Location) by User = Identity
   | where CountOfLocations > 1
-  | extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
+  //| extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
```

Comment:
same feedback on commenting

Reply:
Yeah, this is broken so commented it out while the author fixes it.